SampleProgram/step6.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+MOTOR_EN=/dev/rtmotoren0
+MOTOR_R=/dev/rtmotor_raw_r0
+MOTOR_L=/dev/rtmotor_raw_l0
+COUNTER_R=/dev/rtcounter_r0
+COUNTER_L=/dev/rtcounter_l0
+
+SPEED=400
+
+function echo_counter () {
+    SECONDS=0
+    while [ $SECONDS -lt "$1" ]
+    do
+    echo "count_l:$(cat $COUNTER_L), count_r:$(cat $COUNTER_R)"
+    done
+}
+
+function reset_counters_and_motors () {
+    echo 0 | tee $MOTOR_L $MOTOR_R > /dev/null
+    echo "Reset counter"
+    echo 0 | tee $COUNTER_L $COUNTER_R > /dev/null
+    echo "count_l:$(cat $COUNTER_L), count_r:$(cat $COUNTER_R)"
+}
+
+reset_counter
```

Comment:
`reset_counter`は定義されていないようです。

```suggestion
reset_counters_and_motors
sleep 0.1
```

Reply:
修正しました。